src/views/Details/Details.js

```diff
@@ -26,6 +26,7 @@ export default {
             selectedPayment: null,
             name: null,
             tab: 0,
+            selectedCurrency: "All",
```

Comment:
Probably better to make "All" ⇒  null

Reply:
Just moving  logic to the select, works fine now